api/src/main/java/org/openmrs/api/db/hibernate/HibernateContextDAO.java

```diff
@@ -210,6 +212,20 @@ public User authenticate(String login, String password) throws ContextAuthentica
 		throw new ContextAuthenticationException(errorMsg);
 	}
 	
+	private Long getUnlockTimeMs() {
+		Long unlockTime = TimeUnit.MILLISECONDS.convert(15L, TimeUnit.MINUTES);
+		try
+		{
+			unlockTime = TimeUnit.MILLISECONDS.convert(Long.valueOf(Context.getAdministrationService().getGlobalProperty(
```

Comment:
Isn't it better to first check if StringUtils.isNotBlank for the global property value, and leave the exception to catch invalid non empty values?

Reply:
We've made the change.